src/ResultSet/AbstractResultSet.php

```diff
@@ -33,7 +33,7 @@
     protected $count = null;
 
     /**
-     * @var Iterator|IteratorAggregate|ResultInterface
+     * @var Iterator|IteratorAggregate|ResultInterface|ArrayIterator
```

Comment:
`ArrayIterator` is not really needed - the interfaces cover that

Reply:
You are right, but it is not in my file at https://github.com/fadoe/zend-db/blob/master/src/ResultSet/AbstractResultSet.php
Do you know why it is here?